src/components/AppAlert.svelte

```diff
@@ -3,10 +3,23 @@
   export let bgColor;
```

Comment:
I think we should rethink this component and where/how it's displayed. That can wait until later though.